### ISSUE TITLE
Feature: Make archive/played more clear

### ISSUE THREAD

**Issue by alexstine** [Automattic/pocket-casts-android#749]
### What problem does this address?

Currently, I have no idea what the difference is between marking an episode played or archived. Seems like they both serve the same purpose. Reading from the below docs, archive does everything most would need. Removing the episode from the list, removing the download, etc.

https://support.pocketcasts.com/article/archiving-episodes-2/

### What is your proposed solution?

Explain what marking as played does in docs, in the app, or maybe it is time to remove this feature in favor of archiving. This is all to avoid confusion.

### Did you search for existing feature suggestions?

- [X] I have searched for existing feature suggestions.

**Comment by CookieyedCodes** [comment]
Mark as played = you have listened to the episode
Archive= deletes the episode & hides from the episode list & filters 

I'll agree it's not particularly clear in the notes but their is a benifit to both if you use group by unplayed & the show archive button button archive is basically a listen later function both are veryyyyyy important reason why I love this app, hopefully that clears stuff up tho:)

**Comment by CookieyedCodes** [comment]
@alexstine yup because finishing means you also want to hide the podcast (and deletes it too-in most situations I have starred not archive so my starred don't get deleted-) again check it with grouping it will begin to click ❤️❤️
